### PR TITLE
TOMEE-2891 Translate to Portuguese: examples/resources-jmx-example

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Link to Jira: https://issues.apache.org/jira/browse/TOMEE-2891

### CODE REVIEW

File: examples/resources-jmx-example/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+= Custom resources in an EAR archive
+
+O TomEE permite que você defina seus próprios recursos na sua aplicação e declare-os em `META-INF/resources.xml`. Isso permite que você injete esses recursos em qualquer componente gerenciado na sua aplicação.
+
+Além disso, você também pode definir um método `create` no próprio recurso ou em um POJO que atua como uma fábrica.
+Este exemplo demonstra o uso do método `create` para registrar adicionalmente o recurso como um JMX MBean, além de disponibilizá-lo para injeção.
+
+== Resource
+
+Recursos customizados podem ser definidos usando classes Java muito simples. Nesse caso específico, como o aplicativo também deseja registrar esse como MBean, a classe de recurso precisa seguir o MBean especificação.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:13]:
```suggestion
Recursos customizados podem ser definidos usando classes Java muito simples. Nesse caso específico, como o aplicativo também deseja registrar esse como MBean, a classe de recurso precisa seguir a especificação de MBean.
```

File: examples/resources-jmx-example/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,772 @@
...
+    public void increment();
+
+}
+----
+
+== Método Criar(Create)
+
+To avoid adding the logic to register the resource as an MBean in every
+resource, the application provides a single class with a create() method
+that takes care of this logic for us.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:63]: You forgot to remove this lines.

File: examples/resources-jmx-example/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,772 @@
...
+        this.properties = properties;
+    }
+}
+----
+
+NOTE: que esta classe usa as propriedades definidas na configuração
+(abaixo), combinada com reflexão, para instanciar o recurso e definir
+seus atributos. O código acima requer duas propriedades `code` e `name`
+para saber qual classe criar e o nome JMX para registrá-la
+debaixo.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:153]:
```suggestion
para saber qual classe criar e o nome JMX sob o qual será registrada.
```

File: examples/resources-jmx-example/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,772 @@
...
+    code org.superbiz.resource.jmx.resources.Hello
+    name superbiz.test:name=Hello
+    count 12345
+  </Resource>
+</Resources>
+----
+
+NOTE: que o atributo class-name se refere à classe factory, e não
+o recurso. Depois que o recurso for criado e vinculado ao TomEE,
+Árvore JNDI da fábrica não é mais usada.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:172]:
```suggestion
ao recurso. Depois que o recurso for criado e vinculado à árvore JNDI do TomEE, a fábrica não é mais usada.
```

File: examples/resources-jmx-example/README_pt.adoc [hunk 5]
```diff
@@ -0,0 +1,772 @@
...
+
+Os recursos são normalmente descobertos, criados e vinculados ao
+JNDI muito cedo no processo de implementação, pois outros componentes dependem deles. Isso pode levar a problemas nos quais o classpath final da aplicação ainda não foi determinado e, portanto, o TomEE não pode carregar seu recurso personalizado.
+
+As seguintes propriedades podem ser usadas para alterar esse comportamento.
+
+* Lazy
+
+é um valor booleano que, quando verdadeiro, cria um proxy que adia
+a instanciação real do recurso até a primeira vez que ele olhou para cima do JNDI.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:272]:
```suggestion
a instanciação real do recurso até a primeira vez que ele é procurado pelo JNDI. 
```

File: examples/resources-jmx-example/README_pt.adoc [hunk 6]
```diff
@@ -0,0 +1,772 @@
...
+
+é um valor booleano que, quando verdadeiro, cria um proxy que adia
+a instanciação real do recurso até a primeira vez que ele olhou para cima do JNDI. 
+Isso pode ser útil se o recurso exigir o classpath da aplicação ou para melhorar o tempo de inicialização não totalmente inicializando recursos que podem não ser usados.
+
+* UseAppClassLoader
+
+Esse valor booleano força um recurso instanciado Lazy a usar o
+classloaders da aplicação, em vez do classloader disponível quando o
+os recursos foram processados primeiro.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:279]:
```suggestion
classloaders da aplicação, em vez do classloader disponível quando
os recursos foram processados primeiro.
```

File: examples/resources-jmx-example/README_pt.adoc [hunk 7]
```diff
@@ -0,0 +1,772 @@
...
+
+* InitializeAfterDeployment
+
+Essa configuração booleana força um recurso criado com a propriedade Lazy a ser instanciado depois que a aplicação é iniciado, em vez de esperar que ele seja procurado. Use esse sinalizador se precisar que o recurso seja carregado,independentemente de ser injetado em um componente gerenciado ou consultado manualmente.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:283]:
```suggestion
Essa configuração booleana força um recurso criado com a propriedade Lazy a ser instanciado depois que a aplicação é iniciada, em vez de esperar que ele seja procurado. Use esse sinalizador se precisar que o recurso seja carregado, independentemente de ser injetado em um componente gerenciado ou consultado manualmente.
```

File: examples/resources-jmx-example/README_pt.adoc [hunk 8]
```diff
@@ -0,0 +1,772 @@
...
+Nesse caso, ele definirá esses três sinalizadores para `true`, a menos que o sinalizador `Lazy` tenha sido definido explicitamente.
+
+== PostConstruct / PreDestroy
+
+Como alternativa ao uso de um método de fábrica, você pode usar `@PostConstruct`
+e `@PreDestroy` dentro da sua classe de recurso (observe que você não pode
+usar isso dentro de uma classe de fábrica) para gerenciar qualquer criação ou
+atividades de limpeza. O TomEE chama automaticamente esses métodos quando o
+aplicativo é iniciado e destruído. O uso do `@PostConstruct`
+efetivamente forçar um recurso carregado lentamente a ser instanciado quando o
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:295]:
```suggestion
efetivamente força um recurso carregado lentamente a ser instanciado quando o
```

File: examples/resources-jmx-example/README_pt.adoc [hunk 9]
```diff
@@ -0,0 +1,772 @@
...
+aplicativo está sendo iniciado - da mesma maneira que o a propriedade `InitializeAfterDeployment` faz.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:296]:
```suggestion
aplicativo está sendo iniciado - da mesma maneira que a propriedade `InitializeAfterDeployment` faz.
```

File: examples/resources-jmx-example/README_pt.adoc [hunk 10]
```diff
@@ -0,0 +1,772 @@
...
+INFO: Destroying ProtocolHandler ["http-bio-61309"]
+Apr 15, 2015 12:40:27 PM org.apache.coyote.AbstractProtocol destroy
+INFO: Destroying ProtocolHandler ["ajp-bio-8009"]
+
+Results :
+
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
+----
+
+NOTE:as seguintes linhas mostrando a criação do recurso.
```
Comment (hbelmiro) [inline examples/resources-jmx-example/README_pt.adoc:766]:
```suggestion
NOTE: as seguintes linhas mostrando a criação do recurso.
```

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro for review.

**Pull request merged**